setup.py

```diff
@@ -22,6 +22,7 @@
     python_requires=">=3.7",
     include_package_data=True,
     zip_safe=False,
+    install_requires=["asyncio-mqtt"],
```

Comment:
We should include this as an extra as it's not needed for normal operation.

Example: https://github.com/home-assistant-libs/pytradfri/blob/master/setup.py#L16